components/Navbar.js

```diff
@@ -1,7 +1,87 @@
+import Link from 'next/link'
+import React from 'react'
+import { withRouter } from 'next/router'
 import styled from 'styled-components'
 
-const Navbar = styled.div`
-  border: 1px solid black;
+const InactiveLink = styled.a`
+  text-decoration: none;
+  opacity: ${({ theme }) => theme.transparencies.inactive};
 `
+const ActiveLink = styled.a`
+  text-decoration: none;
+`
+const PrimaryTitle = styled.span`
+  color: ${({ theme }) => theme.colors.primary};
+`
+const PrimaryMutedTitle = styled.span`
+  color: ${({ theme }) => theme.colors.primaryMuted};
+`
+const Subtitle = styled.p`
+  color: ${({ theme }) => theme.colors.text};
+`
+
+const headerSpeak = 'speak.'
+const headerExchange = 'exchange'
+const headerRepeat = 'repeat'
+const exchangeSubheader = 'bilingual dictionary'
+const repeatSubheader = 'vocabulary trainer'
```

Comment:
Is there a reason that these are declared here before they are used, instead of just being used? Like:

```js
const ExchangeTitle = () => (
  <>
    <PrimaryTitle>speak.</PrimaryTitle>
    <PrimaryMutedTitle>exchange</PrimaryMutedTitle>
    <Subtitle>bilingual dictionary</Subtitle>
  </>
)
```